lib/node_modules/@stdlib/stats/base/dists/uniform/cdf/benchmark/benchmark.js

```diff
@@ -58,18 +67,23 @@ bench( pkg+':factory', function benchmark( b ) {
 	var mycdf;
 	var min;
 	var max;
+	var len;
 	var x;
 	var y;
 	var i;
 
 	min = -1.5;
 	max = 1.5;
 	mycdf = cdf.factory( min, max );
+	len = 100;
+	x = new Float64Array(len);
+	for(i = 0; i < len; i++){
+		x[ i ] = uniform(-2.0, 2.0);
```

Comment:
```suggestion
		x[ i ] = uniform( -2.0, 0.0 );
```

Same comments as above